src/openassetio-core/src/hostApi/Manager.cpp

```diff
@@ -300,7 +302,7 @@ void Manager::getWithRelationshipPaged(
     const Manager::BatchElementErrorCallback &errorCallback,
     const trait::TraitSet &resultTraitSet) {
   if (pageSize == 0) {
-    throw std::out_of_range{"pageSize must be greater than zero."};
+    throw errors::InputValidationException{"pageSize must be greater than zero."};
```

Comment:
[plan] Docs need updating (and for below).

Reply:
Addressed as part of #1104